py3status/modules/bluetooth.py

```diff
@@ -3,84 +3,109 @@
 Display bluetooth status.
 
 Configuration parameters:
-    cache_timeout: how often we refresh this module in seconds (default 10)
-    device_separator: the separator char between devices (only if more than one
-        device) (default '|')
-    format: format when there is a connected device (default '{name}')
-    format_no_conn: format when there is no connected device (default 'OFF')
-    format_no_conn_prefix: prefix when there is no connected device
-        (default 'BT: ')
-    format_prefix: prefix when there is a connected device (default 'BT: ')
+    always_collapse: always collapse to hide format_bt (default False)
```

Comment:
I don't like this param I think we can get things working without it.  I'd need to see the other stuff cleaned up first though.